packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -682,6 +683,54 @@ describe('TransactionController', () => {
       expect(controller.state.transactions[0].status).toBe(
         TransactionStatus.unapproved,
       );
+      expect(
+        controller.state.transactions[0]?.dappSuggestedGasFees?.gasPrice,
+      ).toBe('0x1');
+    });
+
+    it('adds dappSuggestedGasFees to transaction if origin is not MM', async () => {
```

Comment:
Should we have similar tests for the below so we explicitly cover all the branches?

- No Origin
- Origin Is MetaMask
- Gas Price
- Max Fee Per Gas & Max Priority Fee Per Gas

We could create a new describe to group all these.

Reply:
Done